scripts/qa_crowbarsetup.sh

```diff
@@ -2365,6 +2365,10 @@ function oncontroller_testsetup()
         fi
     fi
 
+    #test for Glance scrubber service, added after bnc#930739
+    su - glance -s /bin/sh -c "/usr/bin/glance-scrubber --config-dir /etc/glance" || \
+    complain 113 "Glance scrubber doesn't work properly"
```

Comment:
style nitpick

```bash
su ... \
    || complain
```

seems to be the predominant style (and is more readable imho)